assets/src/blocks/ActionsList/index.js

```diff
@@ -8,11 +8,7 @@ export const ACTIONS_LIST_LAYOUT_TYPES = [
 export const registerActionsListBlock = () => {
   const {registerBlockVariation} = wp.blocks;
   const {__} = wp.i18n;
-
-  const IS_NEW_IA = window.p4_vars.options.new_ia;
-  const ACT_PAGE = window.p4_vars.options.take_action_page || -1;
-
-  const queryPostType = IS_NEW_IA ? 'p4_action' : 'page';
+  const postType = 'p4_query_loop_interceptor';
```

Comment:
Since you cannot pass an array of post types from editor, I'd need to pass this custom post type and then intercept it [here](https://github.com/greenpeace/planet4-master-theme/pull/2492/files#diff-5731321646bfd7fe6c8f5582c07ece1a8c4078579f40d518eafc522c4f89f421R37-R39)